lib/sycamore/sycamore/connectors/pinecone/pinecone_writer.py

```diff
@@ -173,15 +172,20 @@ def wait_on_index(client: PineconeGRPC, index: str):
     Takes the name of the index to wait for and blocks until it's available and ready.
     """
     ready = False
+    time.time()
```

Comment:
drop?

Reply:
Dropped the same line in the weaviate test as well